src/components/molecules/definition/definition.jsx

```diff
@@ -7,9 +7,9 @@ import { i18n, splitSentencesToArray } from '../../../utils';
 
 const Definition = ({ results }) => {
 	const payload = results[0];
-	const definitionSentencesArray = splitSentencesToArray(
-		payload.definition.html
-	);
+	const definitionSentencesArray = payload
```

Comment:
should this be payload && payload.definition ?

Reply:
👍

Reply:
Updated, thanks!